acrobat/scripts/scripts.js

```diff
@@ -19,6 +21,17 @@ function loadStyles(paths) {
   });
 }
 
+function addLocale() {
+  const url = new URL(window.location.href);
+  const langFromPath = url.pathname.split('/')[1];
+  const pageLang = localeMap[langFromPath] || 'en-US';
+  const metaTag = document.createElement('meta');
+  metaTag.setAttribute('property', 'og:locale');
+  metaTag.setAttribute('content', pageLang);
+  const head = document.querySelector('head');
```

Comment:
Isn't document.head faster if the code doesn't need to work for older IE (IE < 9)